### PR TITLE
Clicking context menu option toggles calendar

### PR DESCRIPTION
**Pull request by MrAnderson1971**
Clicking context menu option toggles calendar

### CODE REVIEW

**Comment by jaskirat-gill** [comment]
Hey, thank you for the PR. Can you please resolve conflicts?

**Comment by MrAnderson1971** [comment]
> Hey, thank you for the PR. Can you please resolve conflicts?

Done!

File: src/background/index.ts [hunk 1]
```diff
@@ -26,4 +26,19 @@ chrome.action.onClicked.addListener((tab) => {
   }
 });
 
+// Add context menu item
+chrome.contextMenus.create({
+  id: 'workdayCalendar',
+  title: 'Workday Calendar',
```
Comment (Rileyc2022) [inline src/background/index.ts:32]: Maybe it should be a verb like "View Workday Calendar". Up to you though.

Screen Shot 2024-06-07 at 10 17 02 PM

**Review by Rileyc2022 — COMMENTED** [review]
Works for me anyway, good idea I like it.

File: src/background/index.ts (hunk 1 above)
Comment (MrAnderson1971) [reply src/background/index.ts]: Good call. Done!

**Comment by mlool** [comment]
Don't think we need this. Access more permission that we need. And access through context menu doesn't really make it that much easier to access the extension.

**Pull request closed without merging**